### PR TITLE
Adding documentation pages to SQA for the github backup processes for MOOSE and assorted MOOSE apps

### LINKED ISSUE #19252: Automate backups of the MOOSE Github repo
## Reason
Auditing requirements mandate that this is set up

## Design
Cron job to backup the issues/PRs into a csv, gitlab pull mirror for everything else. 

## Impact
Barely interacts with MOOSE itself, should have no impact on functionality, the impact is entirely regulatory.

### PR DESCRIPTION
**Pull request by socratesgorilla**
Closes  #19252

### CODE REVIEW

File: modules/doc/content/sqa/index.md [hunk 1]
```diff
@@ -30,3 +30,5 @@ The documentation for the SQA releases can be found using the following links:
 - [v1.0.0](https://mooseframework.org/releases/moose/v1.0.0)
 
 !sqa report
+
```
Comment (permcody) [inline modules/doc/content/sqa/index.md:33]: Let's move this up above the Table and make a heading for it. I had to look at the source before I finally found the location of this link. It's pretty buried sitting underneath the table. If you haven't looked at the rendered version of your site. Go back to the PR "Conversation" page an look for the MooseBuild comment. It'll post the link to the rendered version of your page so you can see what it looks like.

File: modules/doc/content/sqa/github_backup.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+The issues/PRs backup is accomplished by hitting the Github API for a JSON file of the issues, using internal Python tools to parse the JSON into a dictionary, then dumping the dictionary into a csv file. Backups are maintained for at least two years, after which, older backups may be removed at any time to free up disk space. If the backup is ever needed for a restore on GitHub, the csv can be imported using the Github CSV Tools, available here: https://www.npmjs.com/package/github-csv-tools. If, for whatever reason, the issues need to be introduced to Gitlab, this can be accomplished by simply renaming the 'body' header in the csv to 'description'(the script can similarly be altered to save the csv in the Gitlab format by replacing it in there as well). Importing to Gitlab, however, is strongly not recommended, as the Gitlab issue importer will not be able to import anything from the Github version of the repository except the title and description of the issue, as well as losing every pull request, as Github considers all pull requests to be issues, but Gitlab views them as a separate data structure.
```
[the remark this answers is not in the document]
Comment (permcody) [reply modules/doc/content/sqa/github_backup.md:6]: - Let's go ahead and turn the embedded link into a link (look at the guide for how to do that in Markdown). 
- Also, let's make sure we actually list the internal location of the backups right on this page. Auditors won't consider this as a pass until we either point to a document that mentions to the location, or we point directly to the location. As it stands, an auditor would have to find an human (expert) to find the actual files.

**Review by permcody — CHANGES_REQUESTED** [review]
Getting very close - Let's address the last few comments I've given and then we can get this merged and get the associated Labway issue closed (currently due 11/5). I'd rather not extend it again - Thanks!

File: modules/doc/content/sqa/github_backup.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (GiudGiud) [reply modules/doc/content/sqa/github_backup.md:6]: We strongly recommend against
is not recommended 
is not desirable
all work better

I feel like all these parsers would be written by us if it came down to it. We shouldn't say we will lose everything if we ever need to use that backup

**Review by permcody — APPROVED** [review]
Looks good - Thanks!

**Pull request merged**